specification/Overview.md

```diff
@@ -42,45 +82,58 @@ organised by data category (saveframe).
     a small subset of the RCSB residue variant codes (see below). It remains an
     outstanding issue whether this should be changed or expanded.
 
-    5. 'residue_type' must be specified as the basic type in all cases (e.g. use
+    6. 'residue_name' must be specified as the basic type in all cases (e.g. use
     HIS regardless of protonation state).
 
-    6. A residue is uniquely identified by the 'chain_code' and 'sequence_code',
-    so that the same 'residue_type' string must be used consistently throughout
-    the file. Strictly speaking this makes the 'residue_type' string redundant,
+    7. A residue is uniquely identified by the 'chain_code' and 'sequence_code',
+    so that the same 'residue_name' string must be used consistently throughout
+    the file. Strictly speaking this makes the 'residue_name' string redundant,
     but it is used in identifiers to avoid ambiguity and to serve as a
     cross-check.
 
-    7. Atoms are identified by their name. The stereo/nonstereo assignment
+    8. Atoms are identified by their name. The stereo/nonstereo assignment
     status and atom/atomset/pseudoatom distinction follows from the name, so
     that there is no need for assignment status codes anywhere in the file.
 
-    8. Atoms that differ only by stereochemistry (prochiral protons or methyl
+    9. Atoms that differ only by stereochemistry (prochiral protons or methyl
     groups, NH2 groups, opposite sides of non-rotating aromatic rings) but
     are not stereospecifically assigned, are distinguished using the suffixes
-    'X' and 'Y'.  These suffixes work effectively as a wildcard, so the 'X'
-    and 'Y' stands for one specific digit, and cannot be used for other
-    purposes. The choice of suffix for a given resonance is arbitrary, except
+    'x' and 'y' (lower case).  These suffixes work effectively as a wildcard,
+    standing for 'one or the other stereochemistry specifier'.
+    In most cases 'x' and 'y' stand for one specific digit, but for
+    e.g. DNA/RNA H5' and H5'' the two non-stereospecific names are "H5x" and
+    "H5y", and so stand for either one or two priomes (' or '').
+    The choice of suffix for a given resonance is arbitrary, except
```

Comment:
If stereo specific atoms have degenerate chemical shift / strong overlap, then should it be written as two lines with x and y or  with % in one line?